requirements/common.txt

```diff
@@ -1,8 +1,5 @@
 alembic==0.7.6
-async==0.6.2
```

Comment:
async? For what? I think this was added by Elizabeth by mistake, when she accidentally installed async node module (it was added in the merge request #26). It would've been here earlier if it was used in the production, right?